core/storage/collection/gae_models_test.py

```diff
@@ -80,7 +83,37 @@ def test_get_all_non_private_commits(self):
             collection_models.CollectionCommitLogEntryModel
             .get_all_non_private_commits(2, None, None))
         self.assertEqual(False, commits[2])
-        self.assertEqual('collection--b--0', commits[0][0].id)
+        self.assertEqual('collection-b-0', commits[0][0].id)
+
+    def test_get_all_non_private_commits_with_invalid_max_age(self):
+        with self.assertRaisesRegexp(
+            Exception,
+            'max_age must be a datetime.timedelta instance or None.'):
+            (
+                collection_models.CollectionCommitLogEntryModel
+                .get_all_non_private_commits(2, None, 'invalid_max_age'))
+
+    def test_get_all_non_private_commits_with_max_age(self):
+        private_commit = collection_models.CollectionCommitLogEntryModel.create(
+            'a', 0, 'committer_id', 'username', 'msg',
+            'create', [{}],
+            constants.ACTIVITY_STATUS_PRIVATE, False)
+        public_commit = collection_models.CollectionCommitLogEntryModel.create(
+            'b', 0, 'committer_id', 'username', 'msg',
+            'create', [{}],
+            constants.ACTIVITY_STATUS_PUBLIC, False)
+        private_commit.collection_id = 'a'
```

Comment:
Why do these need to be set manually?

Reply:
Create() does not accept collection_id as a parameter as it is a property of `CollectionCommitLogEntryModel` only. This is why it needs to be set manually.

Reply:
OK -- please add a comment here to explain, and consider pointing to the similar line in CollectionModel._trusted_commit which does this.

Reply:
Done